command/server/config.go

```diff
@@ -494,13 +494,17 @@ func CheckConfig(c *Config, e error) (*Config, error) {
 		return c, e
 	}
 
-	if len(c.Seals) == 2 {
-		switch {
-		case c.Seals[0].Disabled && c.Seals[1].Disabled:
-			return nil, errors.New("seals: two seals provided but both are disabled")
-		case !c.Seals[0].Disabled && !c.Seals[1].Disabled:
-			return nil, errors.New("seals: two seals provided but neither is disabled")
+	if err := c.checkSealConfig(); err != nil {
+		return nil, err
+	}
+
+	sealMap := make(map[string]*configutil.KMS)
+	for _, seal := range c.Seals {
+		if _, ok := sealMap[seal.Name]; ok {
```

Comment:
Honest question, could go either way, should we be case sensitive here or more restrictive?

Reply:
Also, if the seal is not named, will two unnamed seals conflict?

Reply:
Two unnamed seals that have the same type will conflict, since they'll both be given the name of the seal type